app/assets/javascripts/monitorings.js

```diff
@@ -0,0 +1,20 @@
+$('.collapsible > li').click(function () {
+    var id = $(this).data('id');
+    var preloader = "#preloader-" + id;
+
+    $(preloader).show();
+
+    $.getJSON('info/'+ id +'.json', function(data) {
+        $('#text-' + id).text(data['text']);
+        if(data['image']['url']){
+            $('#image-' + id).attr('src', data['image']['url']);
```

Comment:
['image'] is better written in dot notation.<br>['url'] is better written in dot notation.